### PR TITLE
FILTER: Add ChangeAngleRepresentation filter.

### PR DESCRIPTION
**Pull request by imikejackson**
Filter converts angles from Degrees to Radians and vice versa

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>

### CODE REVIEW

File: src/complex/Common/ComplexRange.hpp [hunk 1]
```diff
@@ -45,7 +45,7 @@
 namespace complex
 {
 /**
- * @class ComplexRange ComplexRange.h SIMPLib/Common/ComplexRange.h
+ * @class ComplexRange ComplexRange.h complex/Common/ComplexRange.h
```
Comment (JDuffeyBQ) [inline src/complex/Common/ComplexRange.hpp:48]: This is the only leftover unrelated change I think,

**Comment by imikejackson** [comment]
I don't see that issue on my branch.

**Review by joeykleingers — DISMISSED** [review]
Looks good to me.

**Comment by imikejackson** [comment]
Fixed conflict and ready to merge.

File: src/Plugins/ComplexCore/src/ComplexCore/Filters/ChangeAngleRepresentation.cpp [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    conversionFactor = static_cast<float>(complex::numbers::pi / 180.0f);
+  }
+  else if(pConversionTypeValue == EulerAngleConversionType::RadiansToDegrees)
+  {
+    conversionFactor = static_cast<float>(180.0f / complex::numbers::pi);
+  }
+
+  // Parallel algorithm to find duplicate nodes
+  ParallelDataAlgorithm dataAlg;
+  dataAlg.setRange(0ULL, static_cast<size_t>(angles.getSize()));
```
Comment (JDuffeyBQ) [inline src/Plugins/ComplexCore/src/ComplexCore/Filters/ChangeAngleRepresentation.cpp:141]:
```suggestion
  dataAlg.setRange(0ULL, angles.getSize());
```

**Review by JDuffeyBQ — APPROVED** [review]

**Review by joeykleingers — APPROVED** [review]

File: src/complex/Common/ComplexRange.hpp (hunk 1 above)
Comment (imikejackson) [inline src/complex/Common/ComplexRange.hpp:48]:
```suggestion
 * @class ComplexRange ComplexRange.h complex/Common/ComplexRange.h
```
